content/docs/command-reference/repro.md

```diff
@@ -24,6 +24,8 @@ the dependency graph (a
 by the [stage files](/doc/command-reference/run) (DVC-files with dependencies)
 that are found in the <abbr>project</abbr>. The commands defined in these stages
 can then be executed in the correct order, reproducing pipeline results.
+`dvc repro` relies on the DAG definition that it reads from `dvc.yaml`, and uses
+`dvc.lock` to determine what exactly needs to be run.
```

Comment:
Thanks but this is out of scope for this PR. The whole cmd ref. needs to be rewritten so no use in adding a small update here, probably. Unless it was needed to explain the Dvcfile removal but I don' think that's the case?

Reply:
Ok Sure! will add it later when updating command reference.

Reply:
> See #1572